cups/utils.py

```diff
@@ -14,23 +12,30 @@ def _strtobool(val: str) -> Optional[bool]:
     'val' is anything else.
     """
     val = val.lower()
-    if val in ('y', 'yes', 't', 'true', 'on', '1'):
+    if val in ("y", "yes", "t", "true", "on", "1"):
         return True
-    elif val in ('n', 'no', 'f', 'false', 'off', '0'):
+    elif val in ("n", "no", "f", "false", "off", "0"):
         return False
     else:
         return None
 
+
 def _bytes_to_value(b: bytes) -> Optional[Any]:
     if b == _ffi.NULL:
         return None
     string: str = _ffi.string(b).decode()
     string_to_bool: Optional[bool] = _strtobool(string)
     if isinstance(string_to_bool, bool):
         return string_to_bool
-    if (
-        string
-        and string.split(",")[0] != "none"
-    ):
+    if string and string.split(",")[0] != "none":
         return string
     return None
+
+
+def _value_to_bytes(value: Any) -> Any:
```

Comment:
You could clarify this as 
```suggestion
def _value_to_bytes(value: Optional[Union[list, str]]) -> Any:
```

Reply:
This is kind of useless for a string. Do you think we should use this for strings?